hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/parser/record/transactionhandler/ContractCreateTransactionHandler.java

```diff
@@ -106,6 +106,22 @@ protected void doUpdateEntity(Contract contract, RecordItem recordItem) {
                 break;
         }
 
+        var sidecarRecords = recordItem.getSidecarRecords();
+        for (int i = 0; i < sidecarRecords.size(); i++) {
+            var sidecar = sidecarRecords.get(i);
```

Comment:
Favor enhanced for loops since it reduces the lines of code and avoids the extra get(i).
```suggestion
        for (var sidecar : recordItem.getSidecarRecords()) {
```

Reply:
Thanks I have changed the loop.